src/main/java/mate/academy/spring/dao/MovieSessionDao.java

```diff
@@ -9,4 +9,8 @@ public interface MovieSessionDao extends GenericDao<MovieSession> {
     List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
 
     Optional<MovieSession> get(Long id);
+
+    MovieSession update(MovieSession movieSession);
+
+    void remove(Long id);
```

Comment:
```suggestion
    void delete(Long id);
```
keep to consistency, if you have service method with name delete, then call the same method of DAO layer